frontend/public/locales/en/personal-information.json

```diff
@@ -12,7 +12,8 @@
     "home-address": "Home address",
     "mailing-address": "Mailing address",
     "preferred-language": "Preferred language",
-    "change-addresses": "Change addresses",
+    "change-home-address": "change your home address",
+    "change-mailing-address": "change your mailing address",
```

Comment:
**Nitpick:** Just to be consistent with the buttons we currently have, which start with a capital letter and don't have "your":

```suggestion
    "change-home-address": "Change home address",
    "change-mailing-address": "Change mailing address",
```

This same comment can be applied to `/fr/personal-information.json`.

Reply:
Done

Reply:
FYI it says "Change your home address". 
Conventions from other buttons would mean that it should be "Change home address"